docs/07-outlook/02-learning-goals.adoc

```diff
@@ -25,20 +24,62 @@
   * Abhängig von den Anforderungen und Rahmenbedingungen sollen sie entscheiden können, ob traditionelle Stabilitätsansätze hinreichend sind oder ob Resilient Software Design erforderlich ist.
 - Was sollen die Teilnehmer verstehen?
   * Traditionelle Stabilitätsansätze (Fehlervermeidungsstrategien) auf Infrastrukturebene sind für heutige verteilte, hochvernetzte Systemlandschaften in der Regel nicht mehr hinreichend.
-  * Es gibt keine Silver Bullet für Resilient Software Design, d. h. die relevanten Maßnah- men und eingesetzten Muster und Prinzipien hängen von den Anforderungen, den Rahmenbedingungen und den beteiligten Personen ab.
+  * Es gibt keine Silver Bullet für Resilient Software Design, d. h. die relevanten Maßnahmen und eingesetzten Muster und Prinzipien hängen von den Anforderungen, den Rahmenbedingungen und den beteiligten Personen ab.
 - Was sollen die Teilnehmer kennen?
   * Die Formel für Verfügbarkeit und die unterschiedlichen Ansätze, die Verfügbarkeit zu maximieren (Maximierung von MTTF, Minimierung von MTTR)
-  * Isolation und Latenzüberwachung als sinnvolle Einstiegsprinzipien von Resilient Soft- ware Design
+  * Isolation und Latenzüberwachung als sinnvolle Einstiegsprinzipien von Resilient Software Design
   * Grundlegende Resilience-Muster wie Bulkhead, Circuit Breaker, Redundanz, Failover
 
 // end::DE[]
 
 // tag::EN[]
 [[LG-7-1]]
 ==== LG 7-1: Consistency Models
-tbd.
+- What shall participants be capable of?
```

Comment:
Sometimes you use bullets, sometimes you use
`.What shall participants...`

Should be the same everywhere

Reply:
This will be fixed later. I need to review the whole goals anyway. I will unify them, too.

Reply:
Actually I just fixed it. Was rather minor.